test/smoke/src/main.ts

```diff
@@ -400,7 +401,8 @@ describe(`VSCode Smoke Tests (${opts.web ? 'Web' : 'Electron'})`, () => {
 	setupSearchTests(logger);
 	setupNotebookTests(logger);
 	setupLanguagesTests(logger);
-	if (opts.web) { setupTerminalTests(logger); } // Not stable on desktop/remote https://github.com/microsoft/vscode/issues/146811
+	if (opts.web) { setupTerminalTests(logger); } // Not stable on desktop/remote https://github.com/microsoft/vscode/issues/14681
+	setupTaskTests(logger);
```

Comment:
maybe should only run on web?

Reply:
not sure what the above issue means

Reply:
We don't interact with the terminal so it's probably fine. I'm not sure what was causing the remaining problems for the terminal tests but I doubt they apply to tasks.